components/testing/cu_udpinject/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cu-udpinject"
```

Comment:
cu-udp-inject and cu_udp_inject in project would be better consistant with other packages (and easier to read)

Reply:
done